docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3.6'
+
+services:
+  webpack:
+    container_name: name
+    image: node:16
+    volumes:
+      - ./:/usr/src/app
+    working_dir: /usr/src/app/src/altinn-app-frontend
+    command: bash -c "yarn --immutable && yarn docker"
```

Comment:
Could we instead create a Dockerfile extending from node:16 and install deps/trigger rebuilds when the yarn.lock changes? I love the suggestion in this PR, but I'd like for it to become a first-class citizen instead of something we describe as 'really slow'.. ;-) If you don't want to spend time doing it, I could make these changes after merging this as well.

Reply:
I haven’t (yet) found a great way to do local development in docker. This approach ensures that we always use the latest code on disk, and caches npm packages. A better way would be great!, but my current problem is that we have restrictions on installing software, and when we have docker, there is an obvious solution.